src/main/java/uk/gov/hmcts/reform/sendletter/tasks/SerialTaskRunner.java

```diff
@@ -55,6 +56,11 @@ public void tryRun(int id, Runnable runnable) throws SQLException {
                     }
                 }
             }
+        } catch (SQLException exception) {
+            log.error(
+                String.format("SQL error occurred during task %s run", Task.getTaskFromLockId(id)),
```

Comment:
Let's just pass the Task enum in here instead of ints so we don't need to do this lookup

Reply:
wanted to do so but was in fear in tempering existing implementation